frontend/src/host_orchestrator/main.go

```diff
@@ -153,20 +130,22 @@ func main() {
 		DebugVariablesManager: debugVarsManager,
 	}
 	imController.AddRoutes(r)
-	// The host orchestrator currently has no use for this, since clients won't connect
-	// to it directly, however they probably will once the multi-device feature matures.
-	if len(webUIUrlStr) > 0 {
-		webUIUrl, _ := url.Parse(webUIUrlStr)
-		proxy := httputil.NewSingleHostReverseProxy(webUIUrl)
-		r.PathPrefix("/").Handler(proxy)
-	} else {
-		fs := http.FileServer(http.Dir(DefaultStaticFilesDir))
-		r.PathPrefix("/").Handler(fs)
+
+	if operatorPort == "" {
+		log.Fatal("The host orchestrator requires access to the operator")
 	}
+	operatorURL, err := url.Parse(fmt.Sprintf("http://127.0.0.1:%s", operatorPort))
+	if err != nil {
+		log.Fatalf("Invalid operator port (%s): %v", operatorPort, err)
+	}
+	proxy := httputil.NewSingleHostReverseProxy(operatorURL)
+	// Defer to the operator for every route not covered by the orchestrator. That
+	// includes web UI and other static files.
+	r.PathPrefix("/").Handler(proxy)
```

Comment:
suggest to have this line and `imController.AddRoutes(r)` close to each other, this way we have a very clear location where all the routes are added.

Reply:
Done